packages/azure-services/src/storage/cosmos-container-client.ts

```diff
@@ -170,7 +175,7 @@ export class CosmosContainerClient {
                     break;
                 }
 
-                await System.wait(retryOptions.intervalMilliseconds);
+                await this.sleepFunction(retryOptions.intervalMilliseconds);
```

Comment:
instead, your could make the SystemUtils a class & mock the class.

Reply:
(you don;t have to do that as part of this Pr, if doing that affects many files)